src/Jobbr.DevSupport.StyleCopAnalyzer/Jobbr.DevSupport.StyleCopAnalyzer.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Couldn't we just use a target instead of a whole project?

Reply:
I don't understand what you mean with a "target". Including the files as top-level solution items?

Reply:
Essentially, instead of calling the file `*.csproj` you call it `*.targets` change the `<Project>` tag to not include the SDK definition. Then you can just reference that target in any project that wants to use it with `<Import Project="submodule/devsupport/Jobbr.DevSupport.StyleCopAnalyzer/Jobbr.DevSupport.StyleCopAnalyzer.targets" />`

From a usability point of view nothing really changes, but you also don't get an "empty" project that doesn't actually build anything.

See also: https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-dot-targets-files?view=vs-2022

Reply:
Not familiar with the concept, I'll look into it.

Reply:
It took some wrestling with the project files, but managed to make it work.